Rubberduck.Main/ComClientLibrary/UI/DockableWindowHost.cs

```diff
@@ -429,14 +430,77 @@ private void AdjustSize()
             }
         }
 
+        private static void ToggleDockable(IntPtr hWndVBE)
+        {
+            NativeMethods.SendMessage(hWndVBE, 0x1044, (IntPtr)0xB5, IntPtr.Zero);
```

Comment:
That was more or less a copy-pasta from the original VBA solution from the linked issue, which is also documented (sort of) in the wiki:

https://github.com/rubberduck-vba/Rubberduck/wiki/VBE-Window-Messages

In particular, the `0x1044` is `WM_USER + 3140` and the `0xB5` seems to be a subcode for the `WM_USER + 3140` message. 

My original thought was that this would be a one-trick pony and wouldn't be used anywhere, so it didn't need constants. However, if we want to wrap it in `VbIdeMessage` static class, I can get one started with it just and we can add other when we actually need them.